New-MtrWallpaper.ps1

```diff
@@ -1,353 +1,353 @@
-         $paramCopyItem = @{
-            Path          = ($Path + '\Wallpapers\' + $NewWallpaperName)
-            Destination   = ($Path + '\' + $NewWallpaperFilename)
-            Force         = $true
-            Confirm       = $false
-            ErrorAction   = $CNT
-            WarningAction = $CNT
-         }
-         $null = (Copy-Item @paramCopyItem)
-			
-         # Create the new SkypeSettings.xml using only the fields we need to populate to configure the new wallpaper
-         $paramNewObject = @{
-            TypeName     = 'System.XMl.XmlTextWriter'
-            ArgumentList = (($Path + '\SkypeSettings.xml'), $null)
-         }
-         $xmlWriter = (New-Object @paramNewObject)
-			
-         $xmlWriter.Formatting = 'Indented'
-         $xmlWriter.Indentation = 1
-         $xmlWriter.IndentChar = "`t"
-			
-         $xmlWriter.WriteStartDocument()
-         $xmlWriter.WriteStartElement('SkypeSettings')
-         $xmlWriter.WriteStartElement('Theming')
-         $xmlWriter.WriteElementString('ThemeName', 'Custom')
-         $xmlWriter.WriteElementString('CustomThemeImageUrl', $NewWallpaperFilename)
-         $xmlWriter.WriteStartElement('CustomThemeColor')
-			
-         # Review the Color Settings!
-         $xmlWriter.WriteElementString('RedComponent', $RedComponent)
-         $xmlWriter.WriteElementString('GreenComponent', $GreenComponent)
-         $xmlWriter.WriteElementString('BlueComponent', $BlueComponent)
-			
-         # Close CustomThemeColor
-         $xmlWriter.WriteEndElement()
-			
-         # Close Theming
-         $xmlWriter.WriteEndElement()
-			
-         # Close SkypeSettings
-         $xmlWriter.WriteEndElement()
-			
-         # Save the XML
-         $xmlWriter.WriteEndDocument()
-         $xmlWriter.Flush()
-         $xmlWriter.Close()
-      }
-      #endregion NewWallpaper
-   }
-	
-   end
-   {
-      Write-Verbose -Message ('The new wallpaper is {0}' -f $NewWallpaper)
-   }
-}
-
-New-MtrWallpaper
+﻿function New-MtrWallpaper
+{
+   <#
+         .SYNOPSIS
+         Configure a new wallpaper for Microsoft Teams Room (MTR) System
+
+         .DESCRIPTION
+         Configure a new wallpaper for Microsoft Teams Room (MTR) System.
+
+         This is done by randomly choosing a supported image file from a folder.
+
+         Wallpaper is then copied to the proper folder, and a corresponding XML file is also written.
+
+         When the system reboots as part of its normal scheduled nightly maintenance,
+         the new wallpaper is set for the 'Custom' them in MTR.
+
+         .PARAMETER Path
+         Local MTR User root path, as for now this is the same on every MTR device!
+         Can be a Share (\\<MTRHostname>\\C$\Users\Skype\AppData\Local\Packages\Microsoft.SkypeRoomSystem_8wekyb3d8bbwe\LocalState),
+         if you have more then one MTR or want to do it remotely!
```

Comment:
Admin shares are not enabled on MTRs by default.

Reply:
See above: I never use it.